src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -1040,16 +1040,16 @@ public static string GetXmlSerializerAssemblyName(Type type, string defaultNames
 
         /// <include file='doc\XmlSerializer.uex' path='docs/doc[@for="XmlSerializer.CreateReader"]/*' />
         ///<internalonly/>
-        protected virtual XmlSerializationReader CreateReader() { throw new PlatformNotSupportedException(); }
+        protected virtual XmlSerializationReader CreateReader() { throw new NotImplementedException(); }
```

Comment:
What does NIE mean in this context -- unreachable code (eg ., must always be overridden)